### ISSUE TITLE
Events: error with modules that don't have on_click function

### ISSUE THREAD

**Issue by pelegs** [ultrabug/py3status#1705]
If the user mouse clicks on a module that has no "on_click" function (i.e. sysdata.py, netdata.py) - and error occurs and the user is requested to restart i3.

I tracked the error to line 181 of events.py:
`if module_info["type"] == "py3status":`

On my machine, I put the entire if block into a try/except block, with
`except Exception:
    pass`
and it fixes the issue.

**Comment by lasers** [comment]
Hi, I'm not able to reproduce this. Can you provide more information?
* Version. `py3status --version`. 
* Debug and log this exception. `py3status -d -l /tmp/bottom.log`.

**Comment by pelegs** [comment]
Weird, I try to reproduce this on another machine and it doesn't work either.
Perhaps it's something specific to my machine.

**Comment by lasers** [comment]
Did you find anything new? For what it's worth, I don't think anything changed between `3.12` and latest master so it'd be good to know if you're still experiencing this on latest master. You can log this too by doing  `py3status -l /tmp/py3status.log` and...
```diff
diff --git a/py3status/events.py b/py3status/events.py
index 0a85ddf..6101561 100644
--- a/py3status/events.py
+++ b/py3status/events.py
@@ -177,6 +177,12 @@ class Events(Thread):
         # get the module that the event is for
         module_info = self.output_modules.get(module_name)
 
+        self.py3_wrapper.log('=' * 30)
+        self.py3_wrapper.log(self.output_modules)
+        self.py3_wrapper.log('-' * 30)
+        self.py3_wrapper.log('[{}]'.format(module_info))
+        self.py3_wrapper.log('=' * 30)
+
         # if module is a py3status one call it.
         if module_info["type"] == "py3status":
             module = module_info["module"]

```

**Comment by ultrabug** [comment]
Fixed by #1747 thanks
